app/views/jobs/_form.html.erb

```diff
@@ -0,0 +1,85 @@
+<%= fields_for :job, job do |f|  %>
+  <%= hidden_field_tag "job[generic_type]", @job.generic_type %>
+
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Stellenanzeige</label>
+        <%= f.text_field :title, class: "form-control" %>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Externe ID</label>
+        <%= f.text_field :external_id, class: "form-control" %>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Art der Anstellung</label>
+        <%= text_field_tag "job_payload_employment_type", @job.payload["employmentType"], class: "form-control", name: "job[payload][employment_type]" %>
```

Comment:
[typo] da in der Doku unter https://github.com/ikuseiGmbH/smart-village-app/wiki/Datenmodelle `employement` stand statt `employment` ist das hier nun auch so...lassen oder korrigieren? macht die mobile app schon was mit dem payload content @digorath?

Reply:
mit @digorath geklärt. wird korrekterweise `employment`